strangeQueue.js

```diff
@@ -0,0 +1,36 @@
+import { Stack } from './stack.js';
+
+class StrangeQueue {
+    constructor() {
+        this.stackOne = new Stack();
+        this.stackTwo = new Stack();
+    }
+
+    add(dataUnit) {
+        while (this.stackTwo.data.length !== 0) {
```

Comment:
data should be private member
try to use something like

```js
while (const currentUnit = this.stackTwo.remove()) {
...
}
```

Reply:
changed the method

Reply:
fixed